documentation/modules/ROOT/pages/03_configuration.adoc

```diff
@@ -121,9 +119,9 @@ public class GreetingResourceTest {
 }
 ----
 
-NOTE: If you're still running in live testing mode you will already see that the tests pass again and you can ignore the rest of this page and move on to the next step.
+NOTE: If you're still running in continuous testing mode, you will already see that the tests pass again and you can ignore the rest of this page and move on to the next step.
 
-Stop your current Live Coding session of Quarkus in the terminal by sending a `CTRL+C`:
+Stop your current Live Coding session of Quarkus in the terminal by sending a kbd:[Ctrl+C]:
```

Comment:
Again, we need to make sure this is supported by the tooling.

Reply:
Looks like the kdb: doesn't render correctly: 
![image](https://github.com/redhat-developer-demos/quarkus-tutorial/assets/373537/9deda001-3d77-4597-ae5a-87f26f1845e7)
We'll have to change the 02_basics page back too.

Reply:
OK, that's odd. It's supposed to be supported: https://docs.antora.org/antora/latest/asciidoc/ui-macros/#keybinding and is not new.

I tried to upgrade to latest Antora and same result. I will revert that change and fix this PR too.

Reply:
@kdubois I reverted the kbd change here too.